src/main/webapp/app/entities/programming-exercise.model.ts

```diff
@@ -22,6 +22,8 @@ export enum ProgrammingLanguage {
 export enum ProjectType {
     MAVEN = 'MAVEN',
     ECLIPSE = 'ECLIPSE',
+    PLAIN = 'PLAIN',
+    XCODE = 'XCODE',
```

Comment:
Same here: I would prefer XCODEPROJECT

```suggestion
    XCODE = 'XCODEPROJECT',
```